From 2ee38643b8e29c19b2c6bcef062962460b451754 Mon Sep 17 00:00:00 2001
From: Amir Vadai <avadai@redhat.com>
Date: Mon, 3 Nov 2014 07:52:19 -0500
Subject: [infiniband] iser: Bump driver version to 1.3

Message-id: <bbbb00050196d5151c4c78f95ed83d2df5e0fa61.1415000301.git.avadai@redhat.com>
Patchwork-id: 98690
O-Subject: [Patch RHEL7.1 21/37] IB/iser: Bump driver version to 1.3
Bugzilla: 1107622
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Doug Ledford <dledford@redhat.com>

From: Or Gerlitz <ogerlitz@mellanox.com>

Signed-off-by: Or Gerlitz <ogerlitz@mellanox.com>
Signed-off-by: Roland Dreier <roland@purestorage.com>
(cherry picked from commit 5de2ad986d7e3626e37a024d7ad69dfd6ab13306)
Signed-off-by: Amir Vadai <amirv@mellanox.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/infiniband/ulp/iser/iscsi_iser.h b/drivers/infiniband/ulp/iser/iscsi_iser.h
index a4e4876..324129f 100644
--- a/drivers/infiniband/ulp/iser/iscsi_iser.h
+++ b/drivers/infiniband/ulp/iser/iscsi_iser.h
@@ -69,7 +69,7 @@
 
 #define DRV_NAME "iser"
 #define PFX  DRV_NAME ": "
-#define DRV_VER  "1.1"
+#define DRV_VER  "1.3"
 
 #define iser_dbg(fmt, arg...)    \
  do {      \
-- 
1.7.1